packages/hiro-graph-client/index.d.ts

```diff
@@ -448,6 +448,94 @@ export interface VariablesServlet {
   define<T = any>(options: DefineVariableOptions): Promise<T>;
 }
 
+export type ActionLogType =
+  | 'SearchKI'
+  | 'SearchTask'
+  | 'SearchTeachingSession'
+  | 'DeployKI'
+  | 'UndeployKI'
+  | 'WriteKI'
+  | 'ForkKI'
+  | 'OpenTeachingSession'
+  | 'OwnTeachingSession'
+  | 'StartTeachingSession'
+  | 'ContinueTeachingSession'
+  | 'StartConversionSession'
+  | 'ContinueConversionSession'
+  | 'AssignKI';
+
+export type ActionLogApp = 'KAT' | 'ATQ' | 'Dashboard' | 'KIM' | 'IV';
+
+export interface BaseEvent {
+  action: ActionLogType;
+  desktopApp: ActionLogApp;
+}
+
+export interface SearchEvent extends BaseEvent {
+  searchString: string;
+}
+
+export interface KIEvent extends BaseEvent {
+  kiId: string;
+  kiName: string;
+}
+
+export interface TeachingSessionEvent extends BaseEvent {
+  sessionId: string;
+  sessionName: string;
+  issueId: string;
```

Comment:
shall we use issueId or taskId ?

Reply:
I saw `issueId` here: https://www.pivotaltracker.com/story/show/177142344
Should we change it to `taskId` ?

Reply:
it's issueId in aggregating task https://www.pivotaltracker.com/story/show/177142344
do I need to change it?

Reply:
Please can we change to taskId, I've updated all tickets. Sorry this was my mistake